src/setup/helper_utilities.jl

```diff
@@ -10,3 +10,23 @@ function safe_approx(a, b)
         return a ≈ b
     end
 end
+
+"""
+    is_in_glacier(A::Matrix{F}, distance::I) where {I <: Integer, F <: AbstractFloat}
+
+Return a matrix with booleans indicating if a given pixel is at distance at least
+`distance` in the set of non zero values of the matrix. This usually allows
+discarding the border pixels of a glacier.
+
+Arguments:
+- `A::Matrix{F}`: Matrix from which to compute the matrix of booleans.
+- `distance::I`: Distance to the border, computed as the number of pixels we need
+    to move to find a pixel with value zero.
+"""
+function is_in_glacier(A::Matrix{F}, distance::I) where {I <: Integer, F <: AbstractFloat}
```

Comment:
I've seen this function in the errors which prevent the tests from passing in another PR. It would be important to double check what has been broken in the Halfar solution tests.

Reply:
This is simply because this function didn't existed in the last release, this is the reason CI is now working with it.